src/common/interceptors/errors.interceptor.ts

```diff
@@ -20,7 +21,11 @@ export class ErrorsInterceptor implements NestInterceptor {
             return new HttpException(error.message, HttpStatus.NOT_FOUND)
           }
 
-          if (error instanceof ValidationError) {
+          if (error instanceof ConflictError) {
+            return new HttpException(error.message, HttpStatus.CONFLICT)
```

Comment:
Can't we use Nest-native HttpExceptions here, such as `NotFoundException`, `InternalServerErrorException`, `ConflictException` etc?

Reply:
Sure, as I did before, but later we decided to use `HttpException(error.message, HttpStatus.STATUS)`.
As for me, using Nest-native HttpExceptions is a better way.
Fixed